C04 kks dialogue (#110)
IllusionMods/KKS-Translation: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c04/00/61/translation.txt

//
// Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
//

> 「へぇ、ここではオイル塗りが出来るのね」=“Ehem, we can do oiling here.”

Reviewer: ```suggestion
「へぇ、ここではオイル塗りが出来るのね」=“Huh, so we can put on oil here.”
```
